src/edu/csus/ecs/pc2/ui/team/QuickSubmitter.java

```diff
@@ -92,39 +93,44 @@ public List<File> getAllCDPsubmissionFileNames(IInternalContest mycontest, Strin
     }
 
     /**
-     * submit runs for all input files.
+     * submit runs for all input files.  Guesses language and problem from file path and extension.
      * 
      * Will guess langauge and problem based on path
      * 
      * @see #guessLanguage(IInternalContest, String)
      * @see #guessProblem(IInternalContest, String)
      * 
-     * @param someSubmitFiles
+     * @param a list of files to submit
+     * @return count of files sucessfully submitted/added.
      */
-    public void sendSubmissions(List<File> someSubmitFiles) {
+    public int sendSubmissions(List<File> filesToSubmit) {
 
-        for (File file : someSubmitFiles) {
+        int numberSubmitted = 0;
+        
+        for (File file : filesToSubmit) {
             try {
 
-                Language language = guessLanguage(getContest(), file.getAbsolutePath());
+                Language language = LanguageUtilities.guessLanguage(getContest(), file.getAbsolutePath());
                 if (language == null) {
-                    String ext = getExtension(file.getAbsolutePath());
-                    System.err.println("Cannot identify language for ext= " + ext + " = Can not send submission for file " + file.getAbsolutePath());
+                    String ext = LanguageUtilities.getExtension(file.getAbsolutePath());
+                    log.log(Level.WARNING, "Cannot identify language for ext= " + ext + " = Can not send submission for file " + file.getAbsolutePath());
                 } else {
                     Problem problem = guessProblem(getContest(), file.getAbsolutePath());
                     try {
                         controller.submitJudgeRun(problem, language, file.getAbsolutePath(), null);
-                        System.out.println("submitted run send with language " + language + " and problem " + problem);
+                        log.log(Level.INFO, "submitted run send with language " + language + " and problem " + problem);
```

Comment:
I don't understand the message which is being logged.  "submitted run send with language..." doesn't seem to make sense.  Specifically, the word "send" seems out of place.  Request a change to the message text so that it provides a clearer explanation of what's happening.  (I cannot propose a specific change because, well, as I said to start with, "I don't understand the message...")